### DISCUSSION TITLE
Can I use NNI to train parameters for doc2vec model?

### DISCUSSION THREAD

**Discussion by jexterliangsufe** [microsoft/nni#3759] [Q&A] [answered]
**Environment**:
- NNI version:
- NNI mode (local|remote|pai):
- Client OS:
- Server OS (for remote mode only):
- Python version:
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?:
- Is running in Docker?:

**Log message**:
 - nnimanager.log: 
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**What issue meet, what's expected?**:

**How to reproduce it?**: 

**Additional information**:

**Comment by QuanluZhang** [comment, marked as answer]
yes, you can use NNI to tune hyperparameters of doc2vec model.

**Comment by jexterliangsufe** [comment]
> yes, you can use NNI to tune hyperparameters of doc2vec model.

Thank you!
